panther_description/urdf/gazebo.urdf.xacro

```diff
@@ -6,12 +6,7 @@
 
     <!-- Read property -->
     <xacro:property name="battery_config" value="${xacro.load_yaml(config_file)}" />
-    <xacro:if value="${namespace == ''}">
-      <xacro:property name="ns" value="" />
-    </xacro:if>
-    <xacro:unless value="${namespace == ''}">
-        <xacro:property name="ns" value="${namespace}/" />
-    </xacro:unless>
+    <xacro:property name="ns" value='${namespace + "/" if namespace else ""}' />
```

Comment:
@coderabbitai could you find in our code more patterns like this and make different pull request to apply these changes?

Reply:
Yes, please. Create pull request for that

Reply:
Could you do this on your own?